llm/llama.cpp/generate_linux_cuda.go

```diff
@@ -0,0 +1,17 @@
+package llm
```

Comment:
nice, didnt know separating the file like this would work, will this generate on any linux `go generate` or does it need a flag?

Reply:
It should generate the same as before with the added benefit of selectively generating by explicitly passing the path, e.g. `go generate llm/llama.cpp/generate_linux_cuda.go`.

With `go generate ./...`, it should generate both cpu and cuda